project/AppDependencies.scala

```diff
@@ -61,27 +61,28 @@ object AppDependencies {
   val test: Seq[ModuleID] = PlayCrossCompilation.dependencies(
     shared = Seq(
       "commons-codec"          % "commons-codec"   % "1.12"     % Test,
-      "org.scalatest"          %% "scalatest"      % "3.0.5"   % Test,
+      "org.scalatest"          %% "scalatest"      % "3.1.1"   % Test,
+      "org.scalatestplus"      %% "scalatestplus-mockito" % "1.0.0-M2" % Test,
+      "org.scalatestplus"      %% "scalatestplus-scalacheck"  % "3.1.0.0-RC2" % Test,
       "org.scalacheck"         %% "scalacheck"     % "1.14.0"  % Test,
       "org.pegdown"            % "pegdown"         % "1.6.0"   % Test,
```

Comment:
you can remove pegdown, since the latest scalatest uses flexmark instead

Reply:
Removed, thanks.